style.css

```diff
@@ -841,4 +881,65 @@ textarea::placeholder {
     padding: 0;
     margin: 0;
   }
+
+  /* Modal Projects starts */
+  #modalProject {
+    width: 100%;
+    padding: 0;
+  }
+
+  .modalContent {
+    margin: 5%;
+    gap: unset;
+    width: 100%;
+    height: 100vh;
+  }
+
+  .modalHead {
+    display: flex;
+    justify-content: space-between;
+  }
+
+  .modal-title {
+    font-size: 2.2vw;
+  }
+
+  .modal-tag {
+    font-size: 1vw;
+    line-height: 112%;
+    color: #3c3a39;
+    padding: 1%;
+  }
+
+  #mainBody {
+    display: flex;
+    align-items: flex-start;
+  }
+
+  .modal-description {
+    margin: 3% 0;
+    font-size: 1.1vw;
+    line-height: 130%;
+  }
+
+  .modal-button {
+    padding: 1%;
+    font-size: 1.1vw;
+  }
+
+  .card-img.modal-img {
+    display: flex;
+  }
+
+  #imgModalWork {
+    width: 100%;
+  }
+
+  .modal-side {
+    flex: 1;
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    margin-left: 20px;
+  }
```

Comment:
Great job implementing the details popup window for the desktop view as well, in order to ensure a better UI and good user interaction please implement the following changes for the desktop popup :

- [ ] Your popup view is too big i think you gave it a width that occupies the full screen which is different from what we have in our figma template, kindly reduce the width so it looks closely similar to what we have 😄 
- [ ] Also implement the font color and font type changes for the heading `multi post stories `, use the appropraite font stylings given in the figma
- [ ] Kindly add some spacing between the technologies and the image as shown in the figma template
- [ ] Please fix the padding for the `See live` and `See source` buttons, and also add the image for the `See source` button and add the margin-top. And also reduce the spacing between the two buttons. Kindly make your design to look as close as possible to the Figma design

Figma design :
![image](https://github.com/rubydevi/my-portfolio/assets/20537712/cbbe80b0-fd67-4aad-9758-70deff87220f)

Your design :
![image](https://github.com/rubydevi/my-portfolio/assets/20537712/8115dd80-c43c-4f92-8978-9067882814f9)